roofit/roofitcore/inc/RooFitDriver.h

```diff
@@ -0,0 +1,143 @@
+#ifndef ROO_FIT_DRIVER_H
```

Comment:
Is this header really part of RooFit's interface? Also here, consider moving it to `src/` or `res/`, esp given the many includes that can potentially end up in all RooFit headers through transitive includes...

Reply:
That's correct, the `RooFitDriver`, `RooNLLVarNew`, and `Buffers.h` are pure implementation details! I moved them to `res`.